### DISCUSSION TITLE
Where to find the "Hackathon Graph of Emotions"

### DISCUSSION THREAD

**Discussion by HUSK1E** [github/hackathons#6] [Q&A]
You've probably seen these graphs before:

![c4f75b82-8407-452e-89ba-97663662303b](https://user-images.githubusercontent.com/17418787/118076185-ee84ba00-b3f4-11eb-987e-96c64e999719.jpg)

![PhasesOfFirstYearTeaching](https://user-images.githubusercontent.com/17418787/118076197-f17faa80-b3f4-11eb-983f-1d2923a46807.jpg)

I've seen one like this in regards to the emotions experienced at a hackathon. I think it was from an Atlassian talk. Has anyone seen it anymore and know where to find it?

**Comment by mishmanners** [comment]
I found this one from Atlassian:

![DQPxx7YX0AEBoR_](https://user-images.githubusercontent.com/36594527/118076584-c8134e80-b3f5-11eb-8f33-6fece904dc18.jpg)

I think this is what they usually use to talk about hackathons. There's usually a much higher point at the start however. The person presenting usually describes the emotions as:
- HIGH, yay we're kicking off this thing this is going to be so epic
- MID, OMG we now have to introduce ourselves to people I have no idea
- LOW, what the? We now have to pick something...
- MID, yay, we picked an idea things are going well
- HIGH , YESSS this is going so well, we are killing this
- LOW, OMG how are we going to do this?
- REALLY LOW, WE ARE GONNA DIE, IT'S DUE SOON
- HIGH, we got this, let's crush it
- MID, it's over... I'm now exhausted 🤯 

If anyone finds the EXACT graph, would love to see it.
